src/main/java/net/javadiscord/javabot/data/config/guild/MetricsConfig.java

```diff
@@ -11,6 +11,7 @@
 @Data
 @EqualsAndHashCode(callSuper = true)
 public class MetricsConfig extends GuildConfigItem {
+	private long weeklyMessages = 0;
```

Comment:
What about active members? That's also a field we can't automatically populate like that